source/js/buyers-guide/components/primary-nav/primary-nav.scss

```diff
@@ -144,3 +147,7 @@
     }
   }
 }
+
+#nav-newsletter-form-wrapper.expanded {
+  z-index: 11; /* TODO: is there a var that can be used for this? */
```

Comment:
We use many arbitrary z-index values for Buyer's Guide so I don't know at this point replacing `11` with any [Bootstrap z-index variables](https://getbootstrap.com/docs/4.3/layout/overview/#z-index) can help much?

Reply:
I guess I'll leave it as `11` with a comment to explain why